.gitignore

```diff
@@ -15,3 +15,6 @@ Cargo.lock
 
 # Ignore history files.
 **/.history/**
+
+# Ignore Intellij IDEA files
+/.idea
```

Comment:
Please use your local git ignore to store information like this. You are also allowed to remove `/design`, `.history` from here.

Reply:
okay